### PR TITLE
Initial scaffolding for Draft app

### PR DESCRIPTION
**Pull request by KaylaBrady**
ticket here: https://app.asana.com/0/1199987702860136/1200240419411841

Bulk of code is generated from running `mix phx.new draft`. Made some minor adjustments to make `/assets/js/app.js` => `assets/src/app.tsx` & get the app to compile using typescript based on https://github.com/mbta/arrow.

Confirmed I was able to get the app running locally! 
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
+
+# Since we are building assets from assets/,
+# we ignore priv/static. You may want to comment
+# this depending on your deployment strategy.
+/priv/static/
+
 *.beam
 /config/*.secret.exs
-.elixir_ls/
+.elixir_ls/
```
Comment (paulswartz) [inline .gitignore:38]: can you make sure this (and the other files) have trailing new lines?

File: config/prod.secret.exs [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# In this file, we load production configuration and secrets
```
Comment (paulswartz) [inline config/prod.secret.exs:1]: This is fine for now, but you'll probably want to move this to `config/runtime.exs` at some point: https://hexdocs.pm/elixir/master/Config.html

File: .gitignore (hunk 1 above)
Comment (KaylaBrady) [reply .gitignore]: Updated editor settings & cleaned up those files!

**Review by paulswartz — APPROVED** [review]
🍰

File: config/prod.secret.exs (hunk 2 above)
Comment (KaylaBrady) [reply config/prod.secret.exs]: Deleted this file for now & will update to use runtime as a part of the PR to setup DB -- thanks for pointing to that resource!

**Pull request merged**